.dockerignore

```diff
@@ -1 +1,11 @@
-build
+# Ignore everything
+*
+
+# Allow files and directories
+!/go.mod
+!/go.sum
+!/Makefile
+!/api/**
+!/cmd/**
+!/internal/**
+!/vendor/** 
```

Comment:
uhm... what about Dockerfile.ubi and its dockerignore?

or README and tools.go?

Reply:
Why do we need README, Dockerfile.ubi, and it's dockerignore? Changes to these files should not trigger docker build.

I understand on `tools.go`

Let me know what I am missing?